contrib/kubespray/script/service-boot.sh

```diff
@@ -88,4 +88,13 @@ if [ $? -eq 0 ]; then
   exit 1
 else
   cleanup
+  echo ""
+  echo "OpenPAI is successfully deployed, please check the following information:"
+  echo "Kubernetes cluster config :     ~/pai-deploy/kube/config"
+  echo "OpenPAI cluster config    :     ~/pai-deploy/kube/config"
+  echo "OpenPAI cluster ID        :     pai"
+  echo "Default username          :     admin"
+  echo "Default password          :     admin-password"
+  echo ""
+  echo "You can go to http://<your-master-ip>, then use the default username and password to log in."
```

Comment:
you can get the server address by `kubectl config view -o jsonpath='{.clusters[].cluster.server}'`

Reply:
OK. I changed this to:
```bash
WEBPORTAL_URL=http:$(kubectl config view -o jsonpath='{.clusters[].cluster.server}' | cut -d ":" -f 2)
echo "You can go to ${WEBPORTAL_URL}, then use the default username and password to log in."


```